guides/common/modules/proc_managing-host-packages.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Managing-Host-Packages_{context}"]
+= Managing Host Packages
+
+You can manage the packages of hosts even if the subscriptions are not attached to access repositories.
+
+.Procedure
+. Navigate to *Hosts > All Hosts* and select any Host.
+. Click on the *Content* tab and go to *Packages* where you can see the list of all the available packages with their status.
+
+The Packages pages are paginated, use the arrows to navigate through the pages.
+
+You can *Install packages* available for that particular host by clicking the vertical ellipsis on the top.
```

Comment:
Is this part of the procedure? Should it be part of the list? Maybe reword to ". Click ..."